messages/deploy.metadata.quick.md

```diff
@@ -10,6 +10,8 @@ Executing this quick deploy command takes less time than a standard deploy becau
 
 This command doesn't support source-tracking. The source you deploy overwrites the corresponding metadata in your org. This command doesn’t attempt to merge your source with the versions in your org.
 
+Note: This command is not intended to be used on Sandboxes. By default, Sandboxes do not run tests during a deploy. Use `sf project deploy start` instead.
```

Comment:
```suggestion
Note: Don't use this command on sandboxes.  By default, sandboxes don't run tests during a deploy. Use `sf project deploy start` instead.
```

Reply:
@iowillhoit -- I edited your note, but I'm a little confused about the content.   In particular, I don't see how the fact that sandboxes don't run tests by default during a deploy means that you can't use the validate/quick command pair with sandboxes.  

Can't you run this command on the sanbox:

`sf project deploy validate --test-level RunLocalTests --target-org mysandbox`

And then the quick deploy like this?

`sf project deploy quick --use-most-recent --target-org mysandbox`

Reply:
You can in some situations, but we saw reports of test not running (even though the existed) in a sandbox `validate`. The validate/quick flow is designed for deploys to production deploys. "Confirm the tests are passing, then quickly deploy the validated source". Since sandboxes don't require tests on a deploy, you don't need to use validate/quick. Just use `deploy start` instead.